PROVESFlightControllerReference/test/day-in-the-life/test_day_in_the_life.py

```
@@ -3,21 +3,24 @@
 
-ROOT = Path(__file__).parent.parent
+ROOT = Path(__file__).parent.parent.parent.parent
 
 mode_manager = "ReferenceDeployment.modeManager"
+DOL_SERIAL_PORT = os.environ.get("DOL_SERIAL_PORT", "/dev/ttyACM0")
+
 # Open up the serial port to the ground station
 try:
-    pass_through_serial = serial.Serial("/dev/ttyACM0", baudrate=115200)
+    pass_through_serial = serial.Serial(DOL_SERIAL_PORT, baudrate=115200)
 except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)